wled00/util.cpp

```diff
@@ -66,3 +66,27 @@ uint8_t extractModeName(uint8_t mode, const char *src, char *dest, uint8_t maxLe
   dest[printedChars] = '\0';
   return strlen(dest);
 }
+
+
+CRGB getCRGBForBand(int x, int pal) { 
+  extern int fftResult[];                         // summary of bins array. 16 summary bins.
+  CRGB value;
+  CHSV hsv;
+  if(pal == 71) { // bit hacky to use palette id here, but don't want to litter the code with lots of different methods. TODO: add enum for palette creation type
+    if(x == 0) {
+      value = CRGB(fftResult[10]/2, fftResult[4]/2, fftResult[0]/2);
+    }
+    else if(x == 255) {
+      value = CRGB(fftResult[10]/2, fftResult[0]/2, fftResult[4]/2);
```

Comment:
fftResult[] is int in SR 0.13.x - so better to cast all CRGB parameters to uint8_t

Like `value = CRGB(uint8_t(fftResult[10]/2), uint8_t(fftResult[4]/2), uint8_t(fftResult[0]/2));`